.github/workflows/check-in-tree-build.yml

```diff
@@ -54,12 +54,11 @@ jobs:
           echo "deb https://apt.llvm.org/bionic/ llvm-toolchain-bionic main" | sudo tee -a /etc/apt/sources.list
           echo "deb https://packages.lunarg.com/vulkan bionic main" | sudo tee -a /etc/apt/sources.list
           sudo apt-get update
-          # Linux systems in GitHub Actions already have a pre-installed clang
-          # that is not needed for the translator as we rely on newer version
-          sudo apt-get --purge remove clang-8 clang-9
           sudo apt-get -yq --no-install-suggests --no-install-recommends install \
             clang-${{ env.LLVM_VERSION }} \
             spirv-tools
+          # Linux systems in GitHub Actions already have older versions of clang
+          # pre-installed. Make sure to override these with the relevant version.
           sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-${{ env.LLVM_VERSION }} 1000
```

Comment:
Minor: We might want to use `--set` (manual mode) instead of `--instal` (automatic mode) to make choice of clang even stricter.

Reply:
That's worth a try - would it be fine if I uploaded and tested this in a separate PR?

Reply:
Sure

Reply:
Created https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1043; will open the review after the current PR is merged.